app/resources/playlists/playlists_controller.rb

```diff
@@ -33,6 +33,15 @@ def manifest
     render json: { message: "No manifest found for #{params[:id]}" }
   end
 
+  # View the structural metadata for a given repository resource
+  def structure
+    @change_set = change_set_class.new(find_resource(params[:id])).prepopulate!
+    authorize! :structure, @change_set.resource
+    @logical_order = (Array(@change_set.logical_structure).first || Structure.new).decorate
```

Comment:
Is there any reason for this to be the same as the final instance variable? Couldn't it just be a different local variable?

Reply:
Nah, it's just copied from the ScannedResourcesController.

Reply:
👍